tailwind.config.ts

```diff
@@ -135,4 +134,4 @@ module.exports = {
     },
   },
   plugins: [styleguidePlugin()],
-};
+} as Config;
```

Comment:
Would've liked to use `satisfies` here, but it requires typescript 4.9, which pages doesn't seem to support in the build stage. I'll open an issue.